packages/media/EmbedVideo.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 
-export default function EmbedVideo(videoId) {
+export default function EmbedVideo(props) {
+  const { videoId, className } = props;
   return (
     <iframe
       width="560"
       height="315"
-      src={`https://www.youtube.com/embed/${videoId.videoId}?si=vYU81uVmaV7GSju2&amp;autoplay=1&amp;controls=1&amp;rel=0`}
+      src={`https://www.youtube.com/embed/${videoId}?si=vYU81uVmaV7GSju2&amp;autoplay=1&amp;controls=1&amp;rel=0`}
```

Comment:
@iamrentman Looks like this video is set to autoplay? I seem to remember there was a SODA rule that autoplay videos need to be muted. Might be good to double check 😃